lesson02/assignment/css/_components.content.scss

```diff
@@ -1,17 +1,117 @@
-html {
-    box-sizing: border-box;
+/*style for outter gray box*/
+.main {
+    width: 951px;
+    height: 474px;
```

Comment:
`px` should not be used as an unit of size in modern web development. There are very few exceptions, and this is a cornerstone of responsive design. This card element is being styled to be a stable, non-flexible size, so `rem` would be the correct unit. For larger elements, or those that are part of page layout, `%` is often a more useful unit. Here's an article that explains in more detail: https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984

Reply:
For this assignment, it's fine to leave these as is. The next assignment is to recreate this card element programmatically. Please take that opportunity to update the SCSS to use responsive units of measurement.